sdks/php/.php-cs-fixer.dist.php

```diff
@@ -17,89 +17,19 @@
 ;
 
 $config = new PhpCsFixer\Config();
-$config->setParallelConfig(
-    PhpCsFixer\Runner\Parallel\ParallelConfigFactory::detect()
-);
```

Comment:
You'll want to leave the original contents, to match what Dropbox Sign's codebase uses.